src/ui/components/Viewer.js

```diff
@@ -2,8 +2,13 @@ import React, { useEffect, useState, useRef } from "react";
 import RightSidebar from "./RightSidebar";
 import Tooltip from "./Tooltip";
 import Toast from "./Toast";
+import { installObserver } from "../../protocol/graphics";
 
 export default function Viewer({ tooltip }) {
+  useEffect(() => {
```

Comment:
The viewer is remounted every time we toggle the toolbox collapse. We have to re-attach this observer so that the newly reattached viewer updates on resize.